docs/learn/guide/app.md

```diff
@@ -0,0 +1,74 @@
+# APP Guide
```

Comment:
Can same thing be done with the app guide? 

Instead of separate page on app guide, as soon as one clicks on app guide, they can have a walkthrough.